CHANGELOG.md

```diff
@@ -381,3 +381,14 @@ specified in the `SUPERMETRICS_DEFAULT_USER` secret
 
 ### Fixed
 - Fix `AzureBlobStorage`'s `to_storage()` method is missing the final upload blob part
+
+## [Unreleased] - 2023-04-20
+### Added
+- Sources:
+  - `Supermetrics` 
+- Tests:
+  - unit
+    - `test_supermetrics`
```

Comment:
Don't mention tests in the changelog; the purpose of a changelog is to track changes in a library from the user's perspective.